modules/EvseV2G/iso_server.cpp

```diff
@@ -1193,7 +1207,11 @@ static enum v2g_event handle_iso_charge_parameter_discovery(struct v2g_connectio
 
         if ((req->MaxEntriesSAScheduleTuple_isUsed == (unsigned int)1) &&
             (req->MaxEntriesSAScheduleTuple < res->SAScheduleList.SAScheduleTuple.arrayLen)) {
-            dlog(DLOG_LEVEL_WARNING, "EV's max. SA-schedule-tuple entries exceeded");
+            if (conn->ctx->service_hpc1_enabled) {
```

Comment:
In general, MaxEntriesSAScheduleTuple_isUsed should be ignored in HPC1. Adapt like this